src/daemon/daemon.cpp

```diff
@@ -19,8 +19,10 @@
 #include "base_cloud_init_config.h"
 #include "instance_settings_handler.h"
 #include "runtime_instance_info_helper.h"
+#include "rust/cxx.h"
 #include "snapshot_settings_handler.h"
 
+#include <multipass-petname/src/lib.rs.h>
```

Comment:
IIUC, this is generated, so I would say it is more distant than the usual headers in multipass/, right? 

There could be cases where dependencies go back and forth between rust and C++, but by default I would say that regular C++ header are closer kin than generated ones from the cxx crate. In that sense, I suggest moving this to a separate section, just after all the `<multipass/...>` includes, but before any third party.